governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md

```diff
@@ -0,0 +1,75 @@
+---
+title: SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024
+summary: Onboard USDC ( USD//C on xDai) to Spark Protocol on Gnosis with included parameters.
```

Comment:
1. Where does `USD//C` come from?
2. Change `Spark Protocol` to `SparkLend`
3. Remove empty space after opening parenthesis
4. xDAI vs Gnosis Chain

Reply:
I believe `USD//C on xDai` comes from the Gnosis chain explorer name: https://gnosisscan.io/token/0xddafbb505ad214d7b80b1f830fccc89b60fb7a83
This name might be a bit confusing

Reply:
Correct, VoteWizard. I think `Circle USDC` and `Tether USDT` will be the updated long names unless you disagree.

Reply:
`Circle USDC` and `Tether USDT` look good to me